src/refmvs.rs

```diff
@@ -396,56 +396,28 @@ unsafe extern "C" fn scan_col(
         len = imax(step, cand_bh4);
     };
 }
+
 #[inline]
-unsafe extern "C" fn mv_projection(mv: mv, num: libc::c_int, den: libc::c_int) -> mv {
-    static mut div_mult: [uint16_t; 32] = [
-        0 as libc::c_int as uint16_t,
-        16384 as libc::c_int as uint16_t,
-        8192 as libc::c_int as uint16_t,
-        5461 as libc::c_int as uint16_t,
-        4096 as libc::c_int as uint16_t,
-        3276 as libc::c_int as uint16_t,
-        2730 as libc::c_int as uint16_t,
-        2340 as libc::c_int as uint16_t,
-        2048 as libc::c_int as uint16_t,
-        1820 as libc::c_int as uint16_t,
-        1638 as libc::c_int as uint16_t,
-        1489 as libc::c_int as uint16_t,
-        1365 as libc::c_int as uint16_t,
-        1260 as libc::c_int as uint16_t,
-        1170 as libc::c_int as uint16_t,
-        1092 as libc::c_int as uint16_t,
-        1024 as libc::c_int as uint16_t,
-        963 as libc::c_int as uint16_t,
-        910 as libc::c_int as uint16_t,
-        862 as libc::c_int as uint16_t,
-        819 as libc::c_int as uint16_t,
-        780 as libc::c_int as uint16_t,
-        744 as libc::c_int as uint16_t,
-        712 as libc::c_int as uint16_t,
-        682 as libc::c_int as uint16_t,
-        655 as libc::c_int as uint16_t,
-        630 as libc::c_int as uint16_t,
-        606 as libc::c_int as uint16_t,
-        585 as libc::c_int as uint16_t,
-        564 as libc::c_int as uint16_t,
-        546 as libc::c_int as uint16_t,
-        528 as libc::c_int as uint16_t,
+fn mv_projection(mv: mv, num: libc::c_int, den: libc::c_int) -> mv {
+    static div_mult: [u16; 32] = [
```

Comment:
You probably want to slap a `#[rustfmt::skip]` on here or this nice clean table is going to turn back into a list when we run our rustfmt pass 😅